src/docs/linea.mdx

```diff
@@ -39,7 +39,7 @@ Linea is a ZK-Rollup that targets EVM compatibility. The protocol team centrally
 
 <Section title="General">
 
-    <Parameter name="Block Time" value="3 seconds" tooltip="The rate at which the rollup produces blocks. Keep in mind that the value is subject to change in the future"/>
+    <Parameter name="Block Time" value="2 seconds" tooltip="The rate at which the rollup produces blocks. Keep in mind that the value is subject to change in the future"/>
 
     <Parameter name="Gas Limit" value="61 million" tooltip="The gas limit that can be consumed by an L2 block"/>
```

Comment:
Does the 61 gas limit still apply? they target 24m gas per block maybe the gas limit is modified as-wel

Reply:
The limit is still 61M